src/plot.js

```diff
@@ -119,6 +120,19 @@ export function plot(options = {}) {
 
   figure.scale = exposeScales(scaleDescriptors);
   figure.legend = exposeLegends(scaleDescriptors, options);
+
+  const w = consumeWarnings();
+  if (w > 0) {
+    select(svg).append("text")
+        .attr("x", width)
+        .attr("y", 20)
+        .attr("dy", "-1em")
+        .attr("text-anchor", "end")
+        .text("⚠️")
```

Comment:
```suggestion
        .attr("cursor", "help")
        .text("⚠️")
```

semantically this would feel more appropriate than the text cursor, but it occludes the ⚠️ symbol